static/admin/novius-os/js/jquery.novius-os.js

```diff
@@ -1003,15 +1003,16 @@ define('jquery-nos',
             nosMedia : function(data) {
 
                 data = data || {};
-                var contentUrls = {
-                        'all'   : 'admin/noviusos_media/appdesk?view=media_pick',
-                        'image' : 'admin/noviusos_media/appdesk?view=image_pick'
```

Comment:
I don't see why. It was not possible to change those config lines, so the only thing I did is to put this string into a single var (which still contains the "view" parameter, right before the "current_id").